.vscode/launch.json

```diff
@@ -0,0 +1,22 @@
+// A launch configuration that launches the extension inside a new window
```

Comment:
Does this belong in this PR?

Reply:
@calebporzio I think so. I needed it for development purposes and when creating a new extension using the `yo code` command, it's provided for you. It's also in the repos of popular extensions like this one: https://github.com/Microsoft/vscode-go.

Reply:
K, thanks for the explanation